src/components/TransactionBlock/AssetBridgeTransactionBlock.tsx

```diff
@@ -293,6 +303,7 @@ const AssetBridgeTransactionBlock = ({
       route={availableRoutes?.find((route) => route.id === option.value)}
       isChecked={selectedRoute?.value && selectedRoute?.value === option.value}
       cost={option.extension && `${formatAmountDisplay(option.extension, '$', 2)}`}
+      fees={option?.fees && `${formatAmountDisplay(option.fees, '$', 2)}`}
```

Comment:
`formatAmountDisplay` already returns string

```suggestion
      fees={option?.fees && formatAmountDisplay(option.fees, '$', 2)}
```